yourgov/components/FormMobileFoodVendorPermit/StepActions.tsx

```diff
@@ -19,10 +20,10 @@ export function FormActions() {
 				<Divider />
 				<ButtonGroup>
 					<Button type="submit" variant="primary" loading={isSubmittingStep}>
-						Save and continue
+						{submitText}
 					</Button>
 					<Button
-						type="button"
+						type="submit"
```

Comment:
This doesn't feel right either. Are 2 submit buttons allowed? It perhaps makes the form submit logic easier?

Reply:
I think you'd want two submit buttons since they both submit the form. So long as each browser picks the first one for consistency.